src/modules/Dashboard/features/Participant/Schedule/Schedule.tsx

```diff
@@ -98,7 +101,7 @@ export const ParticipantSchedule = () => {
         >
```

Comment:
I discovered that there's another way to trigger the same "Create Individual Schedule" action:

https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/5276b2d8-0c9a-4a58-8ce7-d070c13999f5

Since the intention of this ticket is to prevent "Guest Account Submission" from creating an Individual Schedule, I think it's sensible to include fixing this in the scope of this ticket. I believe this will do the trick, by updating line 97 above to:

```tsx
          showEditDefaultConfirmation={
            !selectedRespondent.isAnonymousRespondent && !hasIndividualSchedule
          }
```

Reply:
This is a great catch, but rather than modifying `showEditDefaultConfirmation`, I'd suggest updating the `canCreateIndividualSchedule` prop value instead, like so:

```jsx
canCreateIndividualSchedule={!!userId && !selectedRespondent.isAnonymousRespondent}
```

This should prevent the creation of individual schedules entirely, while allowing for the "Edit Default Schedule" confirmation to continue to be shown prior to editing events, which is intended to be shown prior to making changes to the default schedule, like so:

<img width="822" alt="Screenshot 2024-07-04 at 1 19 10 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1670836/5b47363b-1b92-4ccd-993b-9c5ea7229988">

Reply:
Great catch @JustinNusca! Much better solution.

Reply:
Thanks @farmerpaul @JustinNusca!
Agree, let's get this include this within the scope of this ticket given the quick solution and supporting the overall objective. @ionyRojas